_data/people/Joseph87.yml

```diff
@@ -0,0 +1,7 @@
+username: Joseph87
+
+name: Joseph K
+
+Organization: UNAM School of computing 
+
+Why: Jus chiling and learning programming and GiHub
```

Comment:
Thanks for this contribution, @Joseph87. :bow:

I think you've got a small typo in "chiling". That should be "chilling", right? Can you fix that? If so, I'd be happy to merge this pull request.